examples/send_dates_transaction.js

```diff
@@ -0,0 +1,45 @@
+const pkg = require('../package.json');
+const { Exporter } = require('../index')
+const exporter = new Exporter(pkg.name, true)
+
+async function pushData() {
+  const timestamp = Math.floor(new Date() / 1000)
+
+  let lastPosition = await exporter.getLastPosition()
+  console.log(`Last position: ${JSON.stringify(lastPosition)}`)
+  let key = 1
+  if(lastPosition) {
+    key = lastPosition.key + 1
+  }
+
+  console.log("Sending data with transaction...")
+  await exporter.sendDataWithKey({
+    timestamp: timestamp,
+    iso_date: new Date().toISOString(),
+    key: key
+  }, "key")
+
+  let position = {timestamp: timestamp, key: key}
+  console.log(`Saving position: ${JSON.stringify(position)}`)
+  await exporter.savePosition(position)
+
+  let newPosition = await exporter.getLastPosition()
+  console.log(`New position: ${JSON.stringify(newPosition)}`)
```

Comment:
Why is this needed? If there is possible concurrent usage, I think you would want to store the position which you just saved instead of the one returned by `getLastPosition`. If there are no race conditions - can't you just print `position`?

Reply:
I believe the purpose of the example is to illustrate that the 'position' has been successfully stored in Zookeeper.